Dockerfile

```diff
@@ -32,9 +32,13 @@ WORKDIR /staging
 # Copy main executable to staging area
 RUN cp "$(swift build --package-path /build -c release --show-bin-path)/Run" ./
 
-# Uncomment the next line if you need to load resources from the `Public` directory.
+{{#leaf}}
+# Copy any resouces from the public directory and views directory if the directories exist
+RUN [ -d /build/Public ] && { mv /build/Public ./Public && chmod -R a-w ./Public; } || true
+RUN mv /build/Resources ./Resources && chmod -R a-w ./Resources
```

Comment:
Why only check for the existence of `Public`? If you put the check on both directories, you wouldn't even need the `{{#leaf}}` conditional - I would say that the name `Public` is reasonably cautionary enough against people accidentally dropping content in there they didn't want to have show up, so I think it'd be okay to have it uncommented by default in all configurations instead of only Leaf's.

Reply:
(also, as an aside, it annoys me on general principle that the Mustache engine ends up dropping extra newlines into this particular part of the `Dockerfile` in the "Leaf enabled" configuration, so ditching the conditional pleases me aesthetically too 😅)

Reply:
Ok you've convinced me!